test/index.js

```diff
@@ -100,7 +100,7 @@ describe('Drip', function(){
       test
         .set({ account: 1, token: 'x' })
         .identify(helpers.identify())
-        .error('Drip: bad request status=401 msg=Authentication failed, check your credentials', done);
```

Comment:
Yeah no worries - i'm fine with removing since it's not a big deal. Just was surprised I didn't identify it before!